src/less/components/search/index.less

```diff
@@ -1,4 +1,117 @@
 @import (reference) 'mixins';
+@import (reference) './SearchControl';
```

Comment:
@CITguy, let's discuss please.

We are using `.XxxxxControl` and `.XxxxxFacade` for various basic form control mixins, it would be best for us to be consistent.

Reply:
Updated to:

```less
@import (reference) './SearchFacade';
```